dev/tab1tha/k_nn.py

```diff
@@ -0,0 +1,53 @@
+from sklearn.neighbors import KNeighborsClassifier
+import matplotlib.pyplot as plt
+import numpy as np
+
+def k_nearest(data_train, target_train, data_test, target_test):
+    #training model using the k nearest neighbor algorithm
+    #note that 'neighbor' must be spelt without including a 'u' in order for it to work
+    '''This function trains the model then uses it to predict target values of the test data. In it, the accuracy
+    of the K-nearest neighbor algorithm on this data set is computed and returned '''
+    knn = KNeighborsClassifier(n_neighbors = 5)
```

Comment:
What is the motivation for specifying exactly 5 neighbours?
Does the performance change with this parameter?

Please explain in comments, or provide a sub-analysis motivating this decision.

Reply:
Ah! I see that you address this below in `visual_compare`.
Please indicate that `5` is only the default value in a comment.